dev/com.ibm.ws.jbatch.common_fat/src/com/ibm/ws/jbatch/test/BatchAppUtils.java

```diff
@@ -0,0 +1,238 @@
+     * @param packageNames A comma-delimited list of package paths to include in the web application.  
+     *  All path names in each packages will be included unless further filtered by the regex pattern.
+     */
+    static WebArchive buildBatchWar(String appName,
+                                     String regex,
+                                     String... packageNames) {
+
+        WebArchive webApp = ShrinkWrap.create(WebArchive.class, appName);
+        final boolean INCLUDE_SUBPKGS = false;  // Exclude subpackages
+        if (regex == null) {
+            webApp.addPackages(INCLUDE_SUBPKGS, packageNames); // Include all pkg paths
+        } else {
+            webApp.addPackages(INCLUDE_SUBPKGS, Filters.include(regex), packageNames);  // Include only pkg paths matching regex 
+        }
+        // Web-inf resources
+        File webInf = new File("test-applications/" + appName + "/resources/WEB-INF");
+        if (webInf.exists()) {
+            for (File webInfElement : webInf.listFiles()) {
+                if (!!!webInfElement.isDirectory()) { // Ignore classes subdir
+                    webApp.addAsWebInfResource(webInfElement);
+                }
+            }
+        }
+        // Batch job definition files
+        File webInfBatchJobs = new File("test-applications/" + appName + "/resources/WEB-INF/classes/META-INF/batch-jobs");
+        if (webInfBatchJobs.exists()) {
+            for (File batchJob : webInfBatchJobs.listFiles()) {
+                String target = "classes/META-INF/batch-jobs/" + batchJob.getName();
+                webApp.addAsWebInfResource(batchJob, target);
+            }
+        }
+        // Package properties
+        File pkgProps = new File("test-applications/" + appName + "/package.properties");
+        if (pkgProps.exists()) {
+            webApp.addAsWebResource(pkgProps);
+        }
+        // Readme
+        File readme = new File("test-applications/" + appName + "/README.txt");
+        if (readme.exists()) {
+            webApp.addAsWebResource(readme);
+        }
+
+        return webApp;
+    }
+
+    //////////
+    // Methods to deploy applications
+
+    final static String PrebuiltAppArtifactPath = "build/lib/test-application/";  // Relative to autoFVT dir
+
+    public static final String USE_PREBUILT = "USE_PREBUILT";
+
+    static boolean usePrebuilt(String[] options) {
+        return Arrays.asList(options).contains(USE_PREBUILT);
+    }
+
+    static void addDropinPrebuilt(LibertyServer targetServer, String appName) throws Exception{
+        targetServer.copyFileToLibertyServerRoot(PrebuiltAppArtifactPath, "dropins", appName);
+        if (JakartaEE9Action.isActive()) {
+            JakartaEE9Action.transformApp(Paths.get(targetServer.getServerRoot(), "dropins", appName));
+        }
+    }
+
+    public static void addDropinsDbServletAppWar(LibertyServer targetServer, String... options) throws Exception {
+        if (usePrebuilt(options)) {
```

Comment:
From @cgianfra: _I'll need to go over this with you just so I fully understand it. The ShrinkHelper path is what I assumed we'd be using, but I'm not sure when you'd use a prebuilt app instead (and where it would be pre-built from)._ 

For the jbatch.open_fat there's no need to use the support for prebuilt common_fat app archives.  If you choose to keep and use the prebuilt support, here's how you would do it:

```
import static com.ibm.ws.jbatch.test.BatchAppUtils.USE_PREBUILT;

...
   BatchAppUtils.addDropinsDbServletAppWar(server, USE_PREBUILT);
```